sway/config.c

```diff
@@ -587,7 +587,10 @@ bool load_include_configs(const char *path, struct sway_config *config,
 	}
 
 	free(wd);
-	return true;
+
+	char wildcard = path[strlen(path) - 1] == '*';
+	// If not a wildcard path, real path must point to an existing file.
+	return wildcard || file_exists(path);
```

Comment:
-1, pattern could be e.g. `` ./`hostname`.conf `` or `./*.conf`.

Instead, we should check for any `load_include_config` error on line 577.

Reply:
Yep, that's a much better solution. I think I tried that originally but for some reason it didn't resolve the issue. It does now.